AdaptiveCards/sdk/rendering-cards/xamarin/android/render-a-card.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Render a card - Xamarin.Android SDK
+author: almedina-ms
+ms.author: almedina
+ms.date: 12/02/2019
+ms.topic: article
+---
+
+# Render a card - Xamarin.Android
+
+Here's how to render a card using the Xamarin.Android SDK.
+
+## Create Adaptive Card Object Instance from JSON Text
+
+```java
```

Comment:
Shouldn't all these code snippets be C#? By the way, the typical syntax for C# is:

"```cs"

You may want to update all of them to use this syntax for compability with other docs pages